SPECS/kernel-uvm/kernel-uvm.spec

```diff
@@ -137,6 +137,9 @@ cp .config %{buildroot}%{_prefix}/src/linux-headers-%{uname_r} # copy .config ma
 ln -sf "%{_prefix}/src/linux-headers-%{uname_r}" "%{buildroot}/lib/modules/%{uname_r}/build"
 find %{buildroot}/lib/modules -name '*.ko' -exec chmod u+x {} +
 
+mkdir -p %{buildroot}/boot/kernel-uvm/
```

Comment:
variable for kernel-uvm? ${name} or $uname_r} folder?

Reply:
New directory removed and binary placed with vmlinux.bin